src/Service/DynamoDb/src/Result/TableExistsWaiter.php

```diff
@@ -11,18 +11,14 @@
 class TableExistsWaiter extends Waiter
 {
     protected const WAIT_TIMEOUT = 500.0;
-    protected const WAIT_DELAY = 20.0;
+    protected const WAIT_DELAY = 5.0;
```

Comment:
Whereas the default time is 20 seconds, the table are actually created in less that 5 seconds.

Reply:
this code is generated, you can't change it (CI fails)

this change have to be proposed in AWS reposity. Or we should think about a way to override it